src/structures/Guild.js

```diff
@@ -968,22 +967,22 @@ class Guild {
   }
 
   /**
-   * Can be used to overwrite permissions when creating a channel or replacing overwrites.
+   * Overwrites to use when creating a channel or replacing overwrites
    * @typedef {Object} ChannelCreationOverwrites
    * @property {PermissionResolvable} [allow] The permissions to allow
    * **(deprecated)**
    * @property {PermissionResolvable} [allowed] The permissions to allow
    * @property {PermissionResolvable} [deny] The permissions to deny
    * **(deprecated)**
    * @property {PermissionResolvable} [denied] The permissions to deny
-   * @property {RoleResolvable|UserResolvable} id ID of the role or member this overwrite is for
+   * @property {GuildMemberResolvable|RoleResolvable} memberOrRole Member or role this overwrite is for
```

Comment:
Since this is a resolvable I'd put `Member or role resolvable this overwrite is for`